doc/docs/guides/releasing.md

```diff
@@ -0,0 +1,6 @@
+# Making a Release
+1. Tag the HEAD of the release branch and push to github: `git tag <version> && git push --tags`
+
+1. Check version is correct in `_version.py`.
+
+1. Make sure you have `twine` installed and then: `make release`
```

Comment:
@ghickman  why not add twine to the requirements?

Reply:
I considered that but it's only necessary for people who are uploading releases to PyPI so seemed superfluous for typical development.  Your call though!

Reply:
I think this is fine tbh - let's see if it actually trips anyone up :)